include/aca_vlan_manager.h

```diff
@@ -69,9 +69,11 @@ class ACA_Vlan_Manager {
 
   int get_outports_unsafe(uint tunnel_id, string &outports);
 
-  uint get_oam_server_port(uint tunnel_id);
+  void set_aux_gateway(uint tunnel_id, const string auxGateway_id);
 
-  void set_oam_server_port(uint tunnel_id, uint port_number);
+  string get_aux_gateway_id(uint tunnel_id);
+
+  bool is_exist_aux_gateway(const string auxGateway_id);
```

Comment:
aux gateway could mean other gateways like internet gateway, NAT gateway etc. This change is moving in the right direction, just keep in mind we will need to make it more generic later to support owther aux gateways.